scripts/open_url_in_instance.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
Can you add a copyright header, please (similar to the ones in all the other files)?

Reply:
Thank you for reviewing.  Done.